pkgs/development/tools/lv_img_conv/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, pkg-config
+, python3
+, pixman
+, libpng
+, libjpeg
+, librsvg
+, giflib
+, cairo
+, pango
+, nodePackages
+, makeWrapper
+}:
+
+buildNpmPackage rec {
+  pname = "lv_img_conv";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "lvgl";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-LB7NZKwrpvps1cKzRoARHL4S48gBHadvxwA6JMmm/ME=";
+  };
+
+  npmDepsHash = "sha256-uDF22wlL3BlMQ/+Wmtgyjp4CVN6sDnjqjEPB4SeQuPk=";
+
+  nativeBuildInputs = [
+    pkg-config
+    python3
+    makeWrapper
+  ];
+
+  buildInputs = [
+    pixman
+    libpng
+    libjpeg
+    librsvg
+    giflib
+    cairo
+    pango
+    nodePackages.ts-node
```

`ts-node` isn't needed here (and in fact, this is a no-op), but CoreText.framework is for Darwin:
```suggestion
    darwin.apple_sdk.frameworks.CoreText
```